lib/liftoff/xcodeproj_helper.rb

```diff
@@ -79,10 +89,22 @@ def available_targets
       xcode_project.targets.to_a.reject { |t| t.name.end_with?('Tests') }
     end
 
-    def add_shell_script_build_phase(script, name)
+    def add_shell_script_build_phase(script, name, index)
       if build_phase_does_not_exist_with_name?(name)
         build_phase = target.new_shell_script_build_phase(name)
         build_phase.shell_script = script
+
+        # This is not the prettiest way of achieving this but I found no other way
+        # There's probably another way that people that actually know Ruby could think of
+        target.build_phases.delete(build_phase)
+
+        # Sanitize values so that no exception come about
+        if index >= target.build_phases.length || index < -1
+          index = -1
+        end
```

Comment:
This feels overly defensive to me. I don't think we need this check. Trust the user to not be an idiot.